config/config.go

```diff
@@ -635,10 +635,9 @@ func collectsDiff(i1, i2 interface{}, fieldPath string) map[string][]interface{}
 // Load loads config options from a toml file.
 func (c *Config) Load(confFile string) error {
 	metaData, err := toml.DecodeFile(confFile, c)
-	if c.TokenLimit <= 0 {
```

Comment:
The `staticcheck` tool report here?
c.TokenLimit is `uint` and how can it < 0, so the code get the warning.
However, if one day we change the `uint` to `int` alike, the code becomes fragile.

Reply:
Yes, it looks weird isn't it? :-)